.github/workflows/dev-stg-release.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish to the Ballerina Dev\Stage Central
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: choice
+        description: Select Environment
+        required: true
+        options:
+          - DEV CENTRAL
+          - STAGE CENTRAL
+
+jobs:
+  call_workflow:
+    name: Run Dev\Stage Central Publish Workflow
+    if: ${{ github.repository_owner == 'ballerina-platform' }}
+    uses: ballerina-platform/ballerina-library/.github/workflows/dev-stage-central-publish-connector-template.yml@main
+    secrets: inherit
+    with:
+      environment: ${{ github.event.inputs.environment }}
+      additional-publish-flags: '-x test'
```

Comment:
any reason to exclude the test when publishing to stating/dev?

Reply:
Fixed in [564781e](https://github.com/ballerina-platform/module-ballerinax-ai.pgvector/pull/1/commits/564781ef6d23ca98a99816f7eab301d73f16e65f)